src/script/script_error.cpp

```diff
@@ -76,7 +76,7 @@ std::string ScriptErrorString(const ScriptError serror)
         case SCRIPT_ERR_PUBKEYTYPE:
             return "Public key is neither compressed or uncompressed";
         case SCRIPT_ERR_CLEANSTACK:
-            return "Extra items left on stack after execution";
+            return "Stack size must be one after execution";
```

Comment:
since this is trivial change anyways, maybe "must be exactly one", somehow reads better to me

Reply:
Agreed. This sounds better.